packages/sanes-chrome-extension/src/utils/balances/index.ts

```diff
@@ -6,7 +6,7 @@ export type Figures = Omit<Amount, 'tokenTicker'>;
 export function parseFigures(amount: string): Figures {
   // trim off all leading zeros when parsing
   const trimmed = amount.replace(/^0+/, '');
-  const matched = trimmed.match(/^([0-9]+)?([\.\,])?([0-9]+)?$/);
+  const matched = trimmed.match(/^([0-9]+)?([.,])?([0-9]+)?$/);
```

Comment:
Most likely, this diff creates a new bug. `.` means any character while `\.` means dot (which we need here). The `,` does not need to be escaped but the `.` must.

Reply:
Sorry, I was wrong in this case. Just double checked this: inside of a character classe (between `[` and `]`), the `.` litterally means dot. So this change is perfectly fine.